ceph-client.rb

```diff
@@ -37,6 +37,7 @@ class CephClient < Formula
   def install
     ENV.prepend_path "PKG_CONFIG_PATH", "#{Formula["nss"].opt_lib}/pkgconfig"
     ENV.prepend_path "PKG_CONFIG_PATH", "#{Formula["openssl"].opt_lib}/pkgconfig"
+    ENV.prepend_path "PKG_CONFIG_PATH", "/usr/local/lib/pkgconfig"
```

Comment:
note for the posterities and myself, homebrew does not allow add dependencies to cask from a regular (non-cask) formula.